docs/examples/grid_cond_gfn.py

```diff
@@ -253,7 +253,7 @@ def sample_many(self, mbsize):
         return log_ratio
 
     def learn_from(self, it, batch):
-        if type(batch) is list:
+        if isinstance(batch, list):
```

Comment:
Clearly better than using `type`!

Reply:
You can thank `ruff` for catching this 😄